picard/formats/asf.py

```diff
@@ -138,6 +138,7 @@ class ASFFile(File):
         'djmixer': 'WM/DJMixer',
         'mixer': 'WM/Mixer',
         'artists': 'WM/ARTISTS',
```

Comment:
OK, we don't have something like that in quodlibet. We have a "artist" tag containing a list of artists and save it in "Author" as is.